imports/plugins/core/payments/server/no-meteor/schemas/schema.graphql

```diff
@@ -71,9 +70,13 @@ union PaymentMethodData = ExamplePaymentMethodData
 
 "Describes a payment method"
 type PaymentMethod {
```

Comment:
Goal is for fields to always be sorted alpha. (Maybe there is some kind of graphql linter we can use?)